src/main/java/de/zalando/aruha/nakadi/domain/EventTypeStatistics.java

```diff
@@ -0,0 +1,73 @@
+package de.zalando.aruha.nakadi.domain;
+
+import java.util.Objects;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+public class EventTypeStatistics {
+    @NotNull
+    @Min(value = 0, message = "expected_write_rate can't be less then zero")
+    private Integer expectedWriteRate; // messages per minute
```

Comment:
A need of comment here shows that the field name is not informative enough. I suggest to transform comment into field name.
In my opinion the same can be applied to the next field (it can be something like `messageSizeInBytes`)